Source/EventFlow/Extensions/EventFlowOptionsAggregatesExtensions.cs

```diff
@@ -62,7 +64,9 @@ public static class EventFlowOptionsAggregatesExtensions
             this EventFlowOptions eventFlowOptions,
             IEnumerable<Type> aggregateRootTypes)
         {
-            var invalidTypes = aggregateRootTypes
+            var aggregateRootTypeList = aggregateRootTypes.ToList();
```

Comment:
You misses an inline return at Source/EventFlow/Extensions/EventFlowOptionsAggregatesExtensions.cs:59

I will use your style for inline return in future.